src/Management/src/Endpoint/Actuators/HeapDump/HeapDumpEndpointOptions.cs

```diff
@@ -2,16 +2,12 @@
 // The .NET Foundation licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information.
 
-using Steeltoe.Common;
 using Steeltoe.Management.Configuration;
 
 namespace Steeltoe.Management.Endpoint.Actuators.HeapDump;
 
 public sealed class HeapDumpEndpointOptions : EndpointOptions
 {
-    // Default to disabled on Linux + Cloud Foundry until PTRACE is allowed
-    internal override bool DefaultEnabled { get; } = !(Platform.IsCloudFoundry && Platform.IsLinux);
```

Comment:
I'm okay with this change, but why not remove the property entirely?

Reply:
The property is still used in `EndpointOptionsExtensions`, but considering all it does it return `true`... I'm ok with just putting `true` _there_ and deleting the property entirely

Reply:
_Actually..._ before making that change, maybe it _is_ useful to have for custom actuator considerations? Sure, we only had one case in existing Steeltoe code, but it was useful for us to have. So maybe we should leave it?